kiwi/container/oci.py

```diff
@@ -115,7 +115,7 @@ def create(self, filename, base_image):
         """
         exclude_list = Defaults.get_exclude_list_for_root_data_sync()
         exclude_list.append('boot')
-        exclude_list.append('dev')
+        exclude_list.append('dev/*')
         exclude_list.append('sys')
```

Comment:
All of those folders are owned by `filesystem`, so excluding them makes the RPM verification fail.

Could you drop the exclude of `/boot` completely and add `/*` to the others? Are there any hidden files anywhere which `*` wouldn't catch?

Reply:
I did not see hidden files in `/dev`, gonna verify about the others. Yes, I agree we should leave this folders untouched.